apps/portal/js/designer.js

```diff
@@ -597,32 +593,51 @@ $(function () {
                 
                 //maximize logic
                 componentContainer
-                    .addClass('ues-fullview-visible')
-                    .css('height', (designer.height() - 100) + 'px');
-                
-                renderMaxView(component, 'full');
+                    .addClass('ues-component-fullview')
+                    .css('height', ($('#ues-designer').height() - 100) + 'px')
+                    .on('transitionend webkitTransitionEnd oTransitionEnd otransitionend MSTransitionEnd', function() {
+                        componentBody.fadeIn(300);
+                        renderMaxView(component, FULL_COMPONENT_VIEW);
+                        component.fullViewPoped = true;
+                    });
                 
-                component.fullViewPoped = true;
+                componentBody.hide();
             }
         });
-
-        $('body').on('click', '.modal-footer button', function () {
-            $('#componentFull').modal('hide');
-
-        });
-        designer.on('click', '.ues-component .ues-toolbar .ues-properties-handle', function () {
+        
+        designer.on('click', '.ues-component-box .ues-component-toolbar .ues-properties-handle', function () {
             var id = $(this).closest('.ues-component').attr('id');
             renderComponentProperties(findComponent(id));
         });
-        designer.on('click', '.ues-component .ues-toolbar .ues-trash-handle', function () {
-            var id = $(this).closest('.ues-component').attr('id');
-            removeComponent(findComponent(id), function (err) {
-                if (err) {
-                    console.error(err);
-                }
-                saveDashboard();
-            });
+        
+        designer.on('click', '.ues-component-box .ues-component-toolbar .ues-trash-handle', function () {
+            
+            var componentBox = $(this).closest('.ues-component-box'), 
+                id = componentBox.find('.ues-component').attr('id'), 
+                removeBlock = true;
+            
+            if (id) {
+                removeComponent(findComponent(id), function (err) {
+                    if (err) {
+                        removeBlock = false;
+                        console.error(err);
```

Comment:
Removed the console.error. Fixed in the PR https://github.com/wso2/carbon-dashboards/pull/174